.github/workflows/ci.yml

```diff
@@ -1,7 +1,9 @@
 name: Continuous Integration
 
 on: [pull_request]
-
+  paths:
+    - '**.rs'
+    
```

Comment:
```yaml
on:
  pull_request:
    paths:
    - '**.rs'
```

It probably has to be written this way, as `on: [pull_request]` directly assigns to `on` without allowing more children.

Reply:
Alright, that didn't was obvious out of reading the docs.